### ISSUE TITLE
Показалось запутанным использование терминов: "территория", "сообщестово", "территориальное объединение" 

### ISSUE THREAD

**Issue by shlehter** [GetterHubber/WhtN#4]
На сайте и в документе "Тестирование и оценка MVP" часто используются понятия: "территория", "сообщестово", "территориальное объединение". При первом ознакомлении с сайтом и прочтении документа, кажется, что всё это синонимы, не сразу становится ясно, что есть иерархия этих сущностей

При регистрации появляются ещё две сущности "регион" и "район"
![Снимок экрана 2023-02-19 в 17 49 32](https://user-images.githubusercontent.com/124521293/219956124-5d55a686-1d51-4f34-97e2-b92880daec36.png)

![Снимок экрана 2023-02-19 в 17 58 09](https://user-images.githubusercontent.com/124521293/219956145-b01c2c4c-cb2b-4ab1-9713-ed9d0561b171.png)

**Comment by shlehter** [comment]
Ещё попалось слово "Локация"

![Снимок экрана 2023-02-19 в 18 29 45](https://user-images.githubusercontent.com/124521293/219957924-20a74aaf-0a87-4142-a1dd-400b55be66ed.png)

**Comment by GetterHubber** [comment]
Да. Ок! Поработаю. Хорошее слово иерархия! Попробую объяснить через него, может быть.
